spec/views/v1/collections/_collection.json.jbuilder_spec.rb

```diff
@@ -5,12 +5,11 @@
     render partial: 'v1/collections/collection', locals: { collection: Purl.first }
     expect(JSON.parse(rendered)).to include(
       'druid' => 'druid:ee111ff2222',
-      'true_targets' => ['SearchWorksPreview'],
       'catkey' => ''
     )
   end
-  it 'always returns "SearchWorksPreview"' do
-    render partial: 'v1/purls/purl', locals: { purl: Purl.first }
+  it 'always returns "SearchWorksPreview" for non deleted Purls' do
+    render partial: 'v1/collections/collection', locals: { collection: Purl.find(6) }
```

Comment:
Maybe do `Purl.where(deleted_at: nil).first`? Would something like that work? Part of my confusion is that the local variable is for `collection` not `purl` like below, but Purl 6 is an item not a collection object type.